### PR TITLE
Add range of date in full picture and some fix

### PR DESCRIPTION
**Pull request by beudbeud**

### CODE REVIEW

File: prosopopee/themes/material/static/css/styles.css [hunk 1]
```diff
@@ -62,8 +62,8 @@ main {
 }
 
 .bordered-picture .caption__overlay {
-    margin-left: 11.5%;
-    margin-right: 11.5%;
+    margin-left: 11.5% !important;
+    margin-right: 11.5% !important;
```
Comment (Psycojoker) [inline prosopopee/themes/material/static/css/styles.css]: "God css practice" often tell that "!important" is dirty. Is it really needed here?

**Comment by Psycojoker** [comment]
Thanks for this PR :heart: 

Sadly, the most annoying part is missing: ... DOCUMENTATION :fearful: :scream: 

Can you add it plz :p?

**Comment by Psycojoker** [comment]
Oh and documentation for caption is still missing btw :/

**Comment by Psycojoker** [comment]
And updates in CHANGLOG ~o~

**Comment by beudbeud** [comment]
Is done :)

**Comment by Psycojoker** [comment]
Looks great <3

**Pull request merged**